core/dbt/parser/manifest.py

```diff
@@ -426,7 +470,8 @@ def read_saved_manifest(self) -> Optional[Manifest]:
         if os.path.exists(path):
             try:
                 with open(path, 'rb') as fp:
-                    manifest: Manifest = pickle.load(fp)
+                    manifest_mp = fp.read()
+                manifest: Manifest = Manifest.from_msgpack(manifest_mp)  # type: ignore
```

Comment:
If partial parsing is enabled, and we don't find `partial_parse.msgpack`, could we log a message to that effect? Right now, a user can tell because dbt _doesn't_ log `Partial parsing enabled...`, but I'd love to see a message like `Unable to do partial parsing because {path} not found`. (Exact language to be workshopped of course of course)